### PR TITLE
Stories/careplan updates 171057408

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/views/health/services/_service_form.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-= simple_form_for @service, url: @form_url, remote: true do |f|
-  .jServicesForm
+= simple_form_for @service, url: @form_url, remote: request.xhr? do |f|
+  .jBackupPlansForm
```
Comment (awisema) [inline app/views/health/services/_service_form.haml]: Should this be `.jServicesForm`?

File: app/views/health/services/_service_form.haml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
     .row
       .col-sm-8
         = f.input :service_type, collection: @service.class.available_types, input_html: {class: :select2, style: 'width: 100%;'}
```
Comment (awisema) [inline app/views/health/services/_service_form.haml]: The JS below uses `select`, is the correct?

File: app/views/health/services/update.js.coffee [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <% if @service.errors.any? %>
-  $('.jServicesForm .alert.alert-danger').remove()
-  $('.jServicesForm').prepend('<div class="alert alert-danger"><%= @service.errors.full_messages.join(', ') %></div>')
+  $('.jBackupPlansForm .alert.alert-danger').remove()
```
Comment (awisema) [inline app/views/health/services/update.js.coffee]: Should this be `jServicesForm`?

File: app/views/health/services/_service_form.haml (hunk 2 above)
Comment (eanders) [reply app/views/health/services/_service_form.haml]: yes

**Pull request merged**